packages/uswds-tokens/colors/blue-cool.json

```diff
@@ -1,40 +1,26 @@
 {
-  "global": {
-    "category": "system-color-blue-cool"
-  },
-  "props": [
-    {
-      "name": "blue-cool",
-      "value": [
-        { "name": "5", "value": "#e7f2f5" },
-        { "name": "10", "value": "#dae9ee" },
-        { "name": "20", "value": "#adcfdc" },
-        { "name": "30", "value": "#82b4c9" },
-        { "name": "40", "value": "#6499af" },
-        { "name": "50", "value": "#3a7d95" },
-        { "name": "60", "value": "#2e6276" },
-        { "name": "70", "value": "#224a58" },
-        { "name": "80", "value": "#14333d" },
-        { "name": "90", "value": "#0f191c" },
-        {
-          "name": "vivid",
-          "value": [
-            { "name": "5", "value": "#e1f3f8" },
-            { "name": "10", "value": "#c3ebfa" },
-            { "name": "20", "value": "#97d4ea" },
-            { "name": "30", "value": "#59b9de" },
-            { "name": "40", "value": "#28a0cb" },
-            { "name": "50", "value": "#0d7ea2" },
-            { "name": "60", "value": "#07648d" },
-            { "name": "70", "value": "#074b69" },
-            { "name": "80", "value": "#002d3f" },
-            { "name": "90", "value": false }
-          ]
-        }
-      ]
+  "blue-cool": {
+    "5": {"value": "#e7f2f5" },
+    "10": {"value": "#dae9ee" },
+    "20": {"value": "#adcfdc" },
+    "30": {"value": "#82b4c9" },
+    "40": {"value": "#6499af" },
```

Comment:
By default, the token name is set from the key value. From v3 guidance:

> Usually the name for a design token is generated with a [name transform](https://amzn.github.io/style-dictionary/#/transforms?id=transform-types), but you can write your own if you choose. By default Style Dictionary will add a default name which is the key of the design token object.

I don't foresee any naming conflicts since they would be dependent on the nesting structure of the tokens. It does look like we could add a `name` value to the token object to define a specific name if needed!